js/components/controlBar.js

```diff
@@ -119,11 +125,23 @@ var ControlBar = React.createClass({
   closeQualityPopover: function() {
     if(this.props.controller.state.videoQualityOptions.showVideoQualityPopover == true) {
       this.toggleQualityPopover();
+      $("span.oo-icon.oo-icon-bitrate").css("color", this.props.skinConfig.controlBar.iconStyle.inactive.color);
```

Comment:
do not use jQuery with React. Direct DOM manipulation and virtual DOM manipulation do not mix.